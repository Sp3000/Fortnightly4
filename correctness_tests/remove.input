place 0 0 red
count
place 0 1 red
count
place 500 500 red
count
remove 0 0
remove 0 0
count
remove 0 1
count
remove 500 500
count
remove 0 0
halt