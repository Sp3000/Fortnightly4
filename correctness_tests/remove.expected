1
2
3
Error: remove 0 0
2
1
0
Error: remove 0 0